src/Command/Api/ApiCommandHelper.php

```diff
@@ -547,8 +547,6 @@ protected function getSkippedApiCommands(): array {
       'accounts:drush-aliases',
       // Skip any command that has a duplicative corresponding ACLI command.
       'ide:create',
-      'ide:delete',
```

Comment:
This seems like an unrelated (and undesirable) change?

Reply:
Unrelated, yes. Undesirable, no.